a2a/1.0/modules/ROOT/pages/a2a-connector-studio.adoc

```diff
@@ -0,0 +1,138 @@
+
+. Select *General > Authentication* and enter or set the values for these fields:
+. In *Authentication*, select *Expression or Bean reference* and enter the DataWeave expression. 
+. Click *OK*. 
+
+[[basic_authentication]]
+=== Basic Authentication
+
+Basic authentication is a simple (less secure) authentication scheme where the client sends the username and password in plain text (encoded in Base64) in the HTTP Authorization header.
+
+Enter the Basic authentication information in the *Global Element Properties* screen:
+
+. Select *General > Authentication* and enter or set the values for these fields:
+. In *Authentication*, select *Basic authentication* and enter or set the values for these fields:
++
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Username | Username used to initialize the session.
+|Password | Password used to authenticate the user.
+|Preemptive | If set to true, the client sends its authentication credentials (username and password) with the initial HTTP request to the server, rather than waiting for the server to challenge (401 Unauthorized) the client.
+|===
+. Click *OK*.
+
+[[digest]]
+=== Digest
+
+Digest is an HTTP authentication scheme that uses a challenge-response mechanism to avoid sending passwords in plain text. 
+
+Enter the Digest authentication information in the *Global Element Properties* screen:
+
+. Select *General > Authentication* and enter or set the values for these fields:
+. In *Authentication*, select *Digest authentication* and enter or set the values for these fields:
++
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Username | Username used to initialize the session.
+|Password | Password used to authenticate the user.
+|Preemptive | If set to true, the client sends its authentication credentials (username and password) with the initial HTTP request to the server, rather than waiting for the server to challenge (401 Unauthorized) the client.
+|===
+. Click *OK*.
+
+[[ntlm]]
+=== NTLM
+
+NTLM authentication is a Microsoft authentication protocol used primarily in Windows environments. It uses a challenge-response mechanism and is commonly used for network authentication.
+
+Enter the NTLM authentication information in the *Global Element Properties* screen:
+
+. Select *General > Authentication* and enter or set the values for these fields:
+. In *Authentication*, select *Ntlm authentication* and enter or set the values for these fields:
++
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Domain | Name of the proxy domain.
+|Workstation | Name of the proxy workstation.
+|Username | Username used to initialize the session.
+|Password | Password used to authenticate the user.
+|Preemptive | If set to true, the client sends its authentication credentials (username and password) with the initial HTTP request to the server, rather than waiting for the server to challenge (401 Unauthorized) the client.
+|===
+. Click *OK*.
+
```

Comment:
need to add a "Configure other fields" section similar to this, as it is not clickable in the intro steps right now:

https://docs.mulesoft.com/mcp-connector/latest/mcp-connector-studio#configure-other-fields